packaging/package.yml/en.md

```diff
@@ -145,8 +160,10 @@ Macro | Description
 ---- | ----
 **%python_setup** | Runs the build portion of a setup.py using python2.
 **%python_install** | Runs the install portion of a setup.py, to the appropriate root, using python2.
+**%python_compile** | Compiles `*.py` files using python2. This is only useful where the build doesn't compile them already.
```

Comment:
Maybe worth elaborating further? Such as: `doesn't compile them already (indicated by availability of `*.pyc` files).`